crm_rma_stock_location/__init__.py

```diff
@@ -19,5 +21,14 @@
 #
 ##############################################################################
 
-from . import stock_warehouse
-from . import product
+from . import models
+from . import wizards
+from openerp import SUPERUSER_ID
+
+
+def post_init_hook(cr, registry):
```

Comment:
The module adds new location fields to stock.warehouse model, like lot_rma_id in this module, or lot_stock_id in odoo stock module, the locations should be created ever for each warehouse, when this location fields is added in this module, there may be previously warehouses created, then, the location loss_loc_id, lot_refurbish_id and lot_rma_id shoulde be created for these warehouse. https://github.com/odoo/odoo/blob/8.0/addons/stock/stock.py#L3507-L3523